app-containers/aardvark-dns/aardvark-dns-1.8.0.ebuild

```diff
@@ -169,8 +30,12 @@ src_unpack() {
 	fi
 }
 
+src_prepare() {
+	default
+	sed -i -e "s|m0755 bin|m0755 target/$(usex debug debug release)|g;" Makefile || die
+}
+
 src_install() {
-	cargo_src_install
-	dodir /usr/libexec/podman
-	dosym -r /usr/bin/"${PN}" /usr/libexec/podman/"${PN}"
+	export PREFIX="${EPREFIX}"/usr
+	default
```

Comment:
Maybe we should revbump it, because /usr/bin/aardvark-dns no longer is installed, /usr/libexec/podman/aardvark-dns is the real thing rather  a symlink, and now it installs /usr/share/doc/aardvark-dns-1.8.0/README.md.

Reply:
Revbumped in 6ea209549aff585b3ce12b65f3313fec6cf712b0.

Reply:
Looks like it was worth it as well for the debug vs release change.

Reply:
@thesamesam no, that sed line is for src_install(). Previously we were installing using 'cargo_src_install' which would've handled debug properly. Now we're installing using upstream's Makefile so debug is unchanged

Reply:
Another thing to note in the commit message ;)